### ISSUE TITLE
How to tune the altitude loop?

### ISSUE THREAD

**Issue by cnpcshangbo** [cnpcshangbo/simscapeFOPD#7]
![image](https://user-images.githubusercontent.com/4831029/36657291-f853f0f8-1b06-11e8-8221-8efdda1f8911.png)
PID Tuner is not able to tune this plant.

**Comment by cnpcshangbo** [comment]
So try to tune it using my code from FOPD tunner.
By using this project:
https://github.com/cnpcshangbo/FOPD-tunner
I am able to get the optimal IOPID parameters.
C =
 
             1          
  Kp + Ki * --- + Kd * s
             s          

  with Kp = 8.55, Ki = 8.79, Kd = 1.85
 
Continuous-time PID controller in parallel form.


info = 

  struct:

                Stable: 1
    CrossoverFrequency: 7.1642
           PhaseMargin: 83.8783

![image](https://user-images.githubusercontent.com/4831029/36657488-cd4284dc-1b07-11e8-853e-6995cd6ae641.png)

**Comment by cnpcshangbo** [comment]
Then I am going to update those PID parameters in the simulator.
ZPos altitude loop PID continuous Kp = 8.55, Ki = 8.79, Kd = 1.85, Kb=1, ITAE=0.99
![image](https://user-images.githubusercontent.com/4831029/36658012-c3886dd8-1b09-11e8-873e-9c9a3d1bab97.png)
However, the velocity loop has the oscillation.
![image](https://user-images.githubusercontent.com/4831029/36658154-455a8904-1b0a-11e8-8e76-b3bd71e2688f.png)
I have created a new issue for this: https://github.com/cnpcshangbo/simscapeFOPD/issues/8
